README.md

```diff
@@ -12,30 +12,12 @@ your registrations with that additional information.
 This is accomplished by adding a comment directly above the registration starting with `// @knit` and then
 followed with the configuration command.
 
-Commands can be applied at either the Assembly level or the registration level. Options set at the registration level will override any from the Assembly.
-
-### Registration Visibility
-
-Knit can change the visibility of the generated type-safe accessors.
-The following commands are available:
-
-* `public`: Make the type safe function(s) public (default is internal).
-* `internal`: Make the type safe function(s) internal. This is a useful override when the assembly is set to `public`.
-* `hidden`: Do not generate any type safe function.
-* `ignore`: Completely ignore this assembly or registration
-
-#### Examples:
-
-Make the generated type-safe accessor `public`:
-``` swift
-// @knit public
-container.register(MyType.self, factory: { /*...*/ })
-```
+See the [Comment Commands](Docs/CommentCommands.md) documentation for a list of commands.
 
 ### Type Safe Getters
 
 One of the core capabilities of Knit is to generate type safe getters/accessors for your registrations.
-There are some additional options to control and configure this behavior.
+These can be generated in
```

Comment:
Incomplete?

Reply:
Not sure where I was going. Rewrote this sentence.